src/hooks/graphqlState/useProjectState.tsx

```diff
@@ -111,7 +111,9 @@ export const useProjectState = (
       }
 
       if (key === 'links') {
-        input.links = project.links.filter((link) => link)
+        input.links = project.links
+          .filter((link) => link)
```

Comment:
Is there a purpose for the filter here?  Can't you just remove the filter and do a map directly?

Reply:
Yes that is required, user can add extra link, even if it is empty we create an empty record in the project.links array.
- We could stop users from moving ahead, treating empty links as invalid links, then we wouldn't need this.
- But this way, if there is an empty link, the user can still save it, and it'll be saved ignoring the empty link.